littleblog/entry/models.py

```diff
@@ -41,10 +41,10 @@ class Entry(models.Model):
     preview_image = models.ImageField(upload_to='uploads/%Y/%m/%d/', blank=True)
     topic = models.ManyToManyField(Topic, related_name='entries')
     sum_of_marks = models.IntegerField(default=0)
-    current_rate = models.FloatField(default=0)
+    current_rating = models.FloatField(default=0)
```

Comment:
You are changing DB schema here - DB migration file has to be included in PR

Reply:
Hm. migration folder was tracking by git (so I supposed all the files created there would be added), but this file was really untracked.